man/subscription-manager.8

```diff
@@ -604,7 +604,8 @@ option
 .B --remove=NAME
 Removes a named override from the repositories specified with the
 .B --repo
-option
+option.
+Source of the override that is attempted to be removed may be one of the environment(s) the system belongs in. It can only be removed by either: removing the override from the environment it originates from or removing the system from the environment the override originates from.
 
 .TP
 .B --remove-all
```

Comment:
Let's also extend the info on this command, by adding: `... --repo option. Overrides inherited from environments the system belongs in will not be removed. They can only be removed by either: removing them from the environment they originate from or removing the system from those environments.`